src_assets/common/assets/web/config.html

```diff
@@ -391,19 +391,19 @@ <h2 class="accordion-header">
             <pre>tools\dxgi-info.exe</pre>
           </div>
         </div>
-        <div class="mb-3" v-if="platform === 'linux'">
-          <label for="output_name" class="form-label">{{ $t('config.output_name_linux') }}</label>
+        <div class="mb-3" v-if="platform === 'linux' || platform === 'macos'">
+          <label for="output_name" class="form-label">{{ $t('config.output_name_unix') }}</label>
           <input type="text" class="form-control" id="output_name" placeholder="0" v-model="config.output_name" />
           <div class="form-text">
-            {{ $t('config.output_name_desc_linux') }}<br>
+            {{ $t('config.output_name_desc_unix') }}<br>
             <br>
             <pre style="white-space: pre-line;">
```

Comment:
```suggestion
            <pre style="white-space: pre-line;" v-if="platform === 'macos'">
```

> As i said, i think maybe we should consider separate the example in config ui page for linux and macos due to some differences between two platforms.

We can separate macOS and Linux sections by doing this. Then add a second `pre` section for Linux.